components/avtar.css

```diff
@@ -1,17 +1,27 @@
 .avatar-sm img {
```

Comment:
Suggestion: You can use prettier plugin to format the html and css document. It helps in reading code easily.